app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_comment_count
+    post.update(comments_counter: post.comments_counter + 1)
+  end
```

Comment:
- [ ] [Optional] It would be nice to use one of Rails Active Callback methods called `after_save`. It will take a method as an argument and call the method automatically after the model object is saved. After using it, you can decide to make the update_comment_count method a private one:

```ruby on rails 
class Comment < ApplicationRecord
...
after_save :update_comment_count

private

def update_comment_count
    post.update(comments_counter: post.comments.count)
  end
end
````
 
This will make the update automatic and easier. 👍